library/src/main/java/io/github/boguszpawlowski/composecalendar/day/DefaultDay.kt

```diff
@@ -16,6 +18,8 @@ import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.dp
 import io.github.boguszpawlowski.composecalendar.selection.SelectionState
 import java.time.LocalDate
+import java.time.format.TextStyle
+import java.util.*
```

Comment:
This could also be a linter-solvable issue.

For more information on why we should avoid it:
https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb

Personally, I don't think it makes a lot of practical difference. Usually `java.util.*` is added just because IntelliJ decided it's a good and sensible default.

Given that, for a public library I prefer to keep things as explicit as kotlinly possible, so I agree with disallowing wildcard imports